node/src/git.rs

```diff
@@ -34,6 +34,35 @@ pub enum ListRefsError {
     InvalidRef(#[from] RefError),
 }
 
+pub mod refs {
+    use super::*;
+
+    // Where project information is kept.
+    pub const IDENTITY_BRANCH: &str = "radicle/id";
+
+    pub mod storage {
+        use super::*;
+
+        pub fn peer_branch(remote: &RemoteId, branch: &str) -> String {
+            format!("refs/remotes/{remote}/heads/{branch}")
+        }
+
+        pub fn peer_id(remote: &RemoteId) -> String {
+            peer_branch(remote, IDENTITY_BRANCH)
+        }
```

Comment:
I wonder if we should just call these `refs::storage::branch` and `refs::storage::id`, because there's no "local" scope, and `peer_id` for eg. makes it sound like it's the peer's identity, not the project's.